org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/factory/DefaultFeedbackService.java

```diff
@@ -169,4 +181,9 @@ protected void showRequestDialog(byte[] imageBytes) {
 			window.focus();
 		}
 	}
+	
+	static String getFeedbackRecipient(String scValue) {
```

Comment:
this should be private or protected and without the static qualifier.

Reply:
If I remove static, I got following error:
`Cannot make a static reference to the non-static method getFeedbackRecipient(String) from the type DefaultFeedbackService`
for getFeedbackRecipient

Reply:
move the method inside the static class EmailSupportAction